content/community/index.fr.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Communauté Node.js
```

Comment:
Don't forget also to update the title and add the displayTitle field as you did on your other PRs.

Reply:
Bump!

Reply:
Oh s##t